.github/workflows/reuse.yml

```diff
@@ -0,0 +1,254 @@
+    inputs:
+      LEVIATHAN_REF:  # id of input
+        description: "Leviathan branch, or tag, or commit to test with. Examples: 'refs/heads/<branchName>', 'refs/tags/<tagName>', '<commitId>'"
+        required: false
+        type: string
+        default: 'refs/heads/master'
+      # CONFIG:
+      #   description: 'path to config.js'
+      #   required: false
+      #   default: './suites/config.js'
+      SUITES:
+        description: 'path to test folders'
+        required: false
+        type: string
+        default: './suites'
+      # TEST_SUITE_NAME:
+      #   description: 'which test suite to run'
+      #   required: false
+      #   type: string
+      WORKSPACE:
+        description: 'path to WORKSPACE'
+        required: false
+        type: string
+        default: './.leviathan-inputs'
+      DEVICE_TYPE:
+        description: "slug of device type"
+        required: true
+        type: string
+      REPORTS:
+        description: "path to reports directory"
+        required: false
+        type: string
+        default: "./reports"
+      ENVIRONMENT:
+        description: "What BalenaCloud env, balena-cloud or balena-machine"
+        required: true
+        type: string
+        default: "balena-cloud"
+      BALENACLOUD_ORG:
+        required: false
+        description: "Organization used during cloud suite to create fleets, etc."
+        default: "testbot"
+        type: string
+      BALENACLOUD_APP_NAME:
+        description: "Testbot fleet for finding available Leviathan workers. Not used for QEMU workers. Can accept a list of apps separated by commas."
+        type: string
+        required: false
+        default: "balena/testbot-rig"
+      BALENACLOUD_API_URL:
+        type: string
+        required: false
+        default: "balena-cloud.com"
+      BALENAMACHINE_API_URL:
+        required: false
+        type: string
+        default: "bm.balena-dev.com"
+      BALENACLOUD_SSH_URL:
+        required: false
+        type: string
+        default: "ssh.devices.bm.balena-dev.com"
+      BALENACLOUD_SSH_PORT:
+        default: 222
+        type: number
+        required: false
+      QEMU_CPUS:
+        required: false
+        type: number
+        default: 4
+      QEMU_MEMORY:
+        type: string
+        required: false
+        default: "1G"
+      WORKER_TYPE:
+        description: "Valid choices: testbot/qemu"
+        type: string
+        required: true
+      LEVIATHAN_DEBUG:
+        description: |
+          Use this parameter to modify the default debugging settings on test runs. To debug test runs when rebuilding, add new values to this parameter separated with commas. Know more about the possible values available for debugging: https://balena-os.github.io/leviathan/pages/Getting-Started/debugging.html Example value: dev: true, failFast: false
+        type: string
+        required: false
+    secrets:
+      BALENACLOUD_API_KEY:
+        required: false
+      BALENAMACHINE_API_KEY:
```

Comment:
Same as above, we shouldn't have separate inputs for bM, treat it like any other environment.